main/src/cgeo/geocaching/models/Trackable.java

```diff
@@ -38,6 +38,9 @@
     private String type = null;
     @Nullable
     private Date released = null;
+    private Date disposition = null;
+    private String dispositionLogGuid;
+    private String dispositionType;
```

Comment:
I would suggest `logDate`, `logGuid` and `logType` as names (not only here but for the whole PR).

Reply:
Fine idea.